src/ui/public/field_editor/components/field_format_editor/editors/color/color.test.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ColorFormatEditor } from './color';
+import { DEFAULT_COLOR } from '../../../../../../../core_plugins/kibana/common/field_formats/types/color_default';
+
+const fieldType = 'string';
+const format = {
+  getConverterFor: jest.fn(),
+};
+const formatParams = {
+  colors: [{ ...DEFAULT_COLOR }],
+};
+const onChange = jest.fn();
+const onError = jest.fn();
+
+
+describe('ColorFormatEditor', () => {
+  it('should render string type normally (regex field)', async () => {
+    const component = shallow(
+      <ColorFormatEditor
+        fieldType={fieldType}
```

Comment:
Not a big deal, but I'd find these tests easier to read if we just pass in the prop values directly, e.g. "string", "number", etc, instead of creating variables up above.

Reply:
This comment also applies to other tests with a similar pattern.

Reply:
passing on this one - a number of the editors have multiple tests that use the same component props, using the vars reduces duplicate code, particularly for the mock functions (`jest.fn()`) and `format` objects